src/components/Article.jsx

```diff
@@ -1 +1,25 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, text, date }) => (
+
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+
+    <p className="subtitle">
+      <time data-cy="date">
```

Comment:
use dateISO
also, use variables for date formatting as was it App component

Reply:
fixed it all )